scripts/docs.js

```diff
@@ -275,12 +283,74 @@ globby('./lib/*{,/*}.js', { ignore: IGNORE }).then(function(files) {
       }
 
       var docs = parseDocs(file, contents, 0);
-      var outputFile = path.join(
-        './docs/json/master',
-        file.replace('/lib', '').replace('.js', '.json')
-      );
 
-      fs.writeFile(outputFile, JSON.stringify(docs), callback);
+      function writeFile() {
```

Comment:
I think we lose the context of what file is producing the words in that chain. Is there another place I can plug this in that would have that information available?